bayem/visualization.py

```diff
@@ -105,65 +99,80 @@ def visualize_vb_marginal_matrix(
             np.linspace(dists_1d[i].ppf(0.001), dists_1d[i].ppf(0.999), resolution)
         )
 
-        for j in range(i + 1):
-            if focus:
-                axes[i, j].set_xlim(xs[j][0], xs[j][-1])
-
-            if i == j:
-                x = xs[i]
-                axes[i, i].plot(x, dists_1d[i].pdf(x), "-", color=color, lw=lw)
-                if median:
-                    axes[i, i].axvline(
-                        dists_1d[i].median(), ls="--", color=color, lw=lw
-                    )
+        # diagonal plots
```

Comment:
Maybe it would be nice to shortly document what is plotted (what is a diagonal plot?)